app/packages/files/file.ts

```diff
@@ -43,6 +43,10 @@ export interface WaterButlerRevision {
     };
 }
 
+export interface ProviderSpecificData {
+    titleSuffix?: string;
```

Comment:
Suggest `!` over `?` for enforcing the `titleSuffix` because it is an Interface and thereby a contract the Class will inherit.

Reply:
I don't think there is any class inheriting this interface. The way we use `interface` in type-script/ember is to type-check something. In this case, `titleSuffix?: string` is somewhat similar to `titleSuffix: string | undefined`, denoting that `titleSuffix` might not exist in the object. If you do `titleSuffix!: string`, I don't think it compiles because it is a syntax error, as the [definite assignment assertion](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html#definite-assignment-assertions) only works for instance properties or variable assignments, and this is neither.